config/secrets.yml

```diff
@@ -1,5 +1,5 @@
 default: &default
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  secret_key_base: '3c709746a2efc1e2e118cae6d6b86045f70eb1201478aa4166c9be7d830526c90e4cb5320b81085cc1321c3481f01c5fb344b45e3e7dd6d1f1e35a4a8475b0fb'
```

Comment:
This should only be in `development` and/or test, but since `production` and `staging` inherit from `default` this is not a good solution, I don't think.